src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -63,8 +63,10 @@ def populated_devicegraph(planned_devices, disk_names)
         part_creator = Proposal::PartitionCreator.new(original_graph)
         result = part_creator.create_partitions(dist)
 
-        reused.each { |r| r.reuse!(result) }
-        result
+        reused.each { |r| r.reuse!(result.devicegraph) }
```

Comment:
Now the variable name `result` turns out to be confusing. Is not exactly the result of this method anymore.